src/hotspot/share/classfile/classListParser.cpp

```diff
@@ -442,7 +494,7 @@ void ClassListParser::populate_cds_indy_info(const constantPoolHandle &pool, int
       } else if (tag == JVM_CONSTANT_MethodHandle) {
         cii->add_ref_kind(pool->method_handle_ref_kind_at(arg));
         int callee_index = pool->method_handle_klass_index_at(arg);
-        Klass* callee = pool->klass_at(callee_index, THREAD);
+        Klass* callee = pool->klass_at(callee_index, CHECK);
         if (callee != NULL) {
```

Comment:
AFAICS callee can never be NULL if no exception was thrown.

Reply:
Fixed.